container-storage-setup.spec

```diff
@@ -15,10 +15,7 @@ URL:            http://github.com/projectatomic/container-storage-setup/
 Source0: %{git0}/archive/%{commit}/%{repo}-%{shortcommit}.tar.gz
 BuildArch: noarch
 
-BuildRequires:  pkgconfig(systemd)
-
 Requires:       lvm2
-Requires:       systemd-units
```

Comment:
I see that the current package doesn't need to start/restart any unitfiles on install/upgrade. So, we can skip the systemd requirement.